### DISCUSSION TITLE
group-focus does not seem to work on ipad (on safari or edge)

### DISCUSSION THREAD

**Discussion by cynicalduckling** [tailwindlabs/tailwindcss#12945] [Help] [answered]
https://play.tailwindcss.com/B0qMSCCqcw

focusing on the red button should unhide the blue div. it works on windows and android but not on ipad. I dont have a mac so can't really deep dive into the console. Am I doing anything wrong? (this is just for demo, I am using this feature to implement a dropdown menu on navbar)

**Comment by Technobabble17** [comment, marked as answer]
certain browsers don't support focusing on buttons. See: https://stackoverflow.com/questions/42758815/safari-focus-event-doesnt-work-on-button-element 

try swapping the button for a div and giving tab index I believe

**Reply by cynicalduckling** [reply]
Thank you. I will. try that and update you

**Reply by cynicalduckling** [reply]
> certain browsers don't support focusing on buttons. See: https://stackoverflow.com/questions/42758815/safari-focus-event-doesnt-work-on-button-element 
> 
> 
> 
> try swapping the button for a div and giving tab index I believe

That seems to do the trick. Thank you so much :)
